hw/ip/rstmgr/dv/cov/rstmgr_cov_unr_excl.el

```diff
@@ -0,0 +1,437 @@
+// Copyright lowRISC contributors.
```

Comment:
Could you let us know if this is reviewed by designer?

Reply:
Filed https://github.com/lowRISC/opentitan/issues/10694 for the review with Tim.